intro-HPC/leuven/available-modules.tex

```diff
@@ -0,0 +1,102 @@
+\begin{prompt}
```

Comment:
I did not include the whole list and the examples below refer to getting software from 2015a toolchain of from another part of cluster. If you do not like the first example, the list can shorter. But I do not think it should disappear completely… (at least not only from Leuven)

From: Kenneth Hoste [mailto:notifications@github.com]
Sent: donderdag 18 februari 2016 14:07
To: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com>
Cc: Mag Selwa <mag.selwa@kuleuven.be>
Subject: Re: [vsc_user_docs] Feature/leuven chapter1 (#106)


In intro-HPC/leuven/available-modules.tex<https://github.com/hpcugent/vsc_user_docs/pull/106#discussion_r53310601>:

> @@ -0,0 +1,102 @@

> +\begin{prompt}

What's the intention of including this?

Keeping this up-to-date is a nightmare...

Our module tree is updated daily in Ghent...

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/106/files#r53310601>.